extensions/dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -25,13 +25,19 @@ const CodeNameCodeSequenceValues = {
   TrackingUniqueIdentifier: '112040',
   TrackingIdentifier: '112039',
   Finding: '121071',
+  FindingSite: 'G-C0E3', // SRT
+  CornerstoneFreeText: 'CORNERSTONEFREETEXT', // CST4
+};
+
+const CodingSchemeDesignators = {
+  SRT: 'SRT',
```

Comment:
What is "SRT"?

Reply:
Its a DICOM coding scheme designator, its in the standard.